tgui/packages/tgui/interfaces/CrewConsole.js

```diff
@@ -89,23 +96,20 @@ const CrewTable = (props, context) => {
     s => s.ijob
   )(data.sensors ?? []);
   return (
-    <Table>
+    <Table cellpadding="3" >
       <Table.Row>
-        <Table.Cell bold>
+        <Table.Cell bold colspan="2" >
           Name
         </Table.Cell>
-        <Table.Cell bold collapsing />
+        <Table.Cell bold collapsing textAlign="center">
+          Status
+        </Table.Cell>
         <Table.Cell bold collapsing textAlign="center">
           Vitals
         </Table.Cell>
-        <Table.Cell bold textAlign="center">
+        <Table.Cell bold width="180px" collapsing textAlign="center">{/* SKYRAT EDIT - Centers the text*/}
```

Comment:
You're missing a few `// SKYRAT EDIT` stuff in there, which is likely why it was wiped last time.

Reply:
yeah you're right, missed the ones I had before